eurocropsml/acquisition/collector.py

```diff
@@ -47,21 +48,23 @@ def _eolab_finder(
     _, num_days = calendar.monthrange(year, months[1])
     months_list: list[str] = ["0{0}".format(m) if m < 10 else "{0}".format(m) for m in months]
 
-    request_url = """https://finder.eo-lab.org/resto-creodias/api/collections/Sentinel2/search.json\
-?maxRecords={8}&startDate={0}-{1}-01T00%3A00%3A00Z&completionDate={2}-{3}-{4}T23%3A59%3A59Z&cloudCo\
-ver=%5B0%2C{5}%5D&productType=S2MSI{6}&sortParam=startDate&sortOrder=descending&\
-status=all&geometry={7}""".format(
+    request_url = """https://datahub.creodias.eu/odata/v1/Products?$filter=(Attributes/OData.CSC.D\
+oubleAttribute/any(i0:i0/Name eq 'cloudCover' and i0/Value le {0}) and (ContentDate/Start ge {1}-\
+{2}-01T00:00:00.000Z and ContentDate/Start le {1}-{3}-{4}T23:59:59.999Z) and (Online eq true) and \
+(OData.CSC.Intersects(Footprint=geography'SRID=4326;{6}')) and (((((Collection/Name eq 'SENTINEL-2\
+') and not contains(Name, '_N0500') and (((Attributes/Odata.CSC.StringAttribute/any(i0:i0/Name eq \
+'productType' and i0/Value eq 'S2MSI{5}')))))))))&$expand=Attributes&$expand=Assets&$orderby=Conte\
+ntDate/Start asc&$top={7}""".format(
+        max_cloud_cover,
         year,
         months_list[0],
-        year,
         months_list[1],
         num_days,
-        max_cloud_cover,
         processing_level[1:],
         geom_polygon,
         products_per_request,
     )
-
+    print(request_url)
```

Comment:
Is the print intended as user feedback here? Then we should rely on the `logger.info(...)` functionality instead.

If it was just for debugging purposes we should remove it before merging.

Reply:
Sorry, that was still from debugging. I removed it :)